staging/src/k8s.io/kubectl/pkg/cmd/cp/filespec.go

```diff
@@ -98,19 +97,19 @@ func (p remotePath) String() string {
 }
 
 func (p remotePath) Dir() remotePath {
-	return newRemotePath(path.Dir(p.file))
+	return newRemotePath(filepath.Dir(p.file))
 }
 
 func (p remotePath) Base() remotePath {
-	return newRemotePath(path.Base(p.file))
+	return newRemotePath(filepath.Base(p.file))
 }
 
 func (p remotePath) Clean() remotePath {
-	return newRemotePath(path.Clean(p.file))
+	return newRemotePath(filepath.Clean(p.file))
 }
 
 func (p remotePath) Join(elem pathSpec) remotePath {
-	return newRemotePath(path.Join(p.file, elem.String()))
+	return newRemotePath(filepath.Join(p.file, elem.String()))
 }
```

Comment:
cc @soltysh 

I'm pretty sure this change is incorrect... xref https://github.com/kubernetes/kubernetes/pull/94165 ... this is constructing remote paths to be interpreted inside a container, right? if so, the OS kubectl is running on isn't really relevant?

Reply:
Indeed, ``newRemotePath`` will replace ``\`` to ``/`` anyways.